src/main/java/com/health/healther/domain/model/BoardLike.java

```diff
@@ -42,4 +35,11 @@ public class BoardLike extends BaseEntity {
 
 	@Column(name = "IS_LIKED")
 	private boolean isLiked;
+
+	public void deleteBoardLike() {
+		this.isLiked = false;
+	}
+	public void likeBoard() {
+		this.isLiked = true;
+	}
```

Comment:
무엇이 옳은 방법인지는 모르겠지만
```suggestion
	public void controlBoardLike() {
		this.isLiked = isLiked;
	}
```
로 하나만 생성하여 service에서 true, false를 넣어 구현하는 방법도 있다고 생각했습니다~!😁